common/survey/_survey/surveyDependencies.js

```diff
@@ -48,8 +48,29 @@ const buildGraph = survey =>
 
 const getDependencyGraph = R.propOr({}, keys.dependencyGraph)
 
-const getNodeDefDependencies = (nodeDefUuid, dependencyType) =>
-  R.path([keys.dependencyGraph, dependencyType, nodeDefUuid])
+const getNodeDefDependencies = (nodeDefUuid, dependencyType = null) =>
+  R.ifElse(
+    R.always(R.isNil(dependencyType)),
+    //return all node def dependents
+    R.pipe(
+      R.values,
+      R.reduce((accDependents, graph) =>
+          R.pipe(
+            R.propOr([], nodeDefUuid),
+            R.ifElse(
```

Comment:
is it possible to use ```R. unless``` here?

Reply:
no because it won't return the accumulator (accDependents)